openmc/lattice.py

```diff
@@ -865,7 +865,7 @@ def create_xml_subelement(self, xml_element, memo=None):
         # Export the Lattice outer Universe (if specified)
         if self._outer is not None:
             outer = ET.SubElement(lattice_subelement, "outer")
-            outer.text = '{0}'.format(self._outer._id)
+            outer.text = f'{self._outer._id}'
```

Comment:
This probably ought to just be
```suggestion
            outer.text = str(self._outer._id)
```